infrastructure/crypto/Cargo.toml

```diff
@@ -1,5 +1,14 @@
 [package]
 name = "crypto"
 version = "0.0.1"
+edition = "2018"
 
 [dependencies]
+digest = "0.7.6"
+rand = "0.5.5"
+ed25519-dalek = { version = "0.8.1"}
+sha2 = "0.7.1"
```

Comment:
This should be in sync with the github.com/dalek-cryptography/ed25519-dalek dependency version

Reply:
Same as above. "^0.7" resolves to "0.7.1"